DSE_6.8_Release_Notes.md

```diff
@@ -2,6 +2,34 @@
 DSE 6.8.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
 
+# Release notes for 6.8.23
+11 May 2022
+
+## Components versions for DSE 6.8.23
+ * Apache Solr™ 6.0.1.4.2919&ast;
+ * Apache Spark™ 2.4.0.18
+ * Apache TinkerPop™ 3.4.5-20220405-a52bbe2c
+ * Apache Tomcat® 8.5.75
+ * DSE Java Driver 1.10.0-dse+20210424
+ * Netty 4.1.34.3.dse
+ * Spark JobServer 0.8.0.50
+
+## 6.8.23 DSE Core
+* Ensure that sstables are not unnecessarily read for columns that are not selected. See CASSANDRA-16737. (Previously DB-4974). (DSP-22478)
```

Comment:
The same as for 5.1.31:
If we want to stick to the guideline (https://datastax.jira.com/wiki/spaces/EN/pages/3374972929/Release+notes+guideline), the sentence should start with the verb in the past tense.
How about sth like: Improved reading logic to ensure sstables are not unnecessarily read for columns that are not selected?